### PR TITLE
Expose Webapp.last_updated date in the API (bug 1127055)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=1127055

### CODE REVIEW

File: mkt/webapps/tests/test_serializers.py [hunk 1]
```diff
@@ -40,6 +42,10 @@ def serialize(self, app, profile=None):
         a = AppSerializer(instance=app, context={'request': self.request})
         return a.data
 
+    def test_base(self):
```
Comment (robhudson) [inline mkt/webapps/tests/test_serializers.py]: `test_base` -- did you intend to add more stuff here for testing?

**Comment by ngokevin** [comment]
r+

**Comment by robhudson** [comment]
r+

File: mkt/webapps/tests/test_serializers.py (hunk 1 above)
Comment (diox) [inline mkt/webapps/tests/test_serializers.py]: Sort of. I thought it was pretty useless to have a lot of different tests for basic fields, so I introduced this method with the hope that future basic tests would go in it :)

**Pull request merged**